pennylane/plugins/default_gaussian.py

```diff
@@ -795,42 +785,36 @@ def apply(self, operation, wires, par):
 
         self._state = [means, cov]
 
-    def expand_one(self, S, wire):
-        r"""Expands a one-mode Symplectic matrix S to act on the entire subsystem.
+    def expand(self, S, wires):
+        r"""Expands a Symplectic matrix S to act on the entire subsystem.
 
         Args:
-            S (array): :math:`2\times 2` Symplectic matrix
-            wire (int): the wire S acts on
+            S (array): a :math:`2M\times 2M` Symplectic matrix
+            wires (Sequence[int]): the wires of modes S acts on
 
         Returns:
             array: the resulting :math:`2N\times 2N` Symplectic matrix
         """
-        S2 = np.identity(2*self.num_wires)
+        if self.num_wires == 1:
+            # total number of wires is 1, simply return the matrix
+            return S
 
-        ind = np.concatenate([np.array([wire]), np.array([wire])+self.num_wires])
-        rows = ind.reshape(-1, 1)
-        cols = ind.reshape(1, -1)
-        S2[rows, cols] = S.copy()
+        N = self.num_wires
+        w = np.asarray(wires)
 
-        return S2
+        if np.any(w < 0) or np.any(w >= N) or len(set(w)) != len(w):
```

Comment:
would the user be able to sneak non-`int` wires values in here?

Reply:
I don't think so, there is validation in the `Operation` class to catch this.